src/index.ts

```diff
@@ -1,20 +1,5 @@
-import log from "npmlog";
-
 import server from "./server";
-import getConfig from "./config";
-
-let config;
-
-try {
-  config = getConfig();
-} catch (e) {
-  log.error("setup", e.message);
-  process.exit(1);
-}
+import config from "./config";
```

Comment:
this shouldnt be included in the programmatic part 🤔 

the current implementation of `config` is "get stuff from the environment" - which is only feasable in cli usage. instead we should implement some kind of 'programmatic config gatherer' or just export the `IServerConfig` Interface & `server` method.

actually I'm not even sure if this file is needed at all… instead we could just use `server.ts` directly for programatic usage (like we do in `cli.ts` 🤔 )